src/workerd/api/http-test.wd-test

```diff
@@ -8,11 +8,24 @@ const unitTests :Workerd.Config = (
           ( name = "worker", esModule = embed "http-test.js" )
         ],
         bindings = [
-          ( name = "SERVICE", service = "http-test" )
+          ( name = "SERVICE", service = "http-test" ),
+          ( name = "cacheEnabled", text = "false" ),
```

Comment:
The name you chose here uses an inconsistent style from the immediate preceding line.

Please make an effort to be consistent with surrounding code, even in the absence of specific rules.

```suggestion
          ( name = "CACHE_ENABLED", text = "false" ),
```